metron-stellar/stellar-common/src/main/antlr4/org/apache/metron/stellar/common/generated/Stellar.g4

```diff
@@ -254,6 +259,7 @@ identifier_operand :
   | EXISTS LPAREN IDENTIFIER RPAREN #ExistsFunc
   | LPAREN conditional_expr RPAREN #condExpr_paren
   | functions #func
+  | DEFAULT #Default
```

Comment:
I just noticed this. Why do we have default here? This would allow default to be used in some unintended places. Is there some reasoning behind this?